lib/Gateway.js

```diff
@@ -47,20 +54,36 @@ function loadCustomDevices (devices) {
   )
 }
 
+var lastCustomDevicesLoad = 0
+function tryLoadCustomDevices (file, first) {
+  if (Date.now() - lastCustomDevicesLoad < 1000) {
+    return
+  }
+
+  try {
+    const fileContent = fs.readFileSync(file)
+    loadCustomDevices(JSON.parse(fileContent))
+    lastCustomDevicesLoad = Date.now()
+  } catch (error) {
+    if (first) {
```

Comment:
where is `first` var declaration?

Reply:
It's on line 58; it's an argument to `tryLoadCustomDevices`: https://github.com/OpenZWave/Zwave2Mqtt/pull/675/files/2a659f216c24c2d877e80d31623da12654f39f27#diff-cf3d598357eac50f80be79bac63d057aR58

Reply:
Sorry missed it :)